src/averageAges.js

```diff
@@ -55,7 +68,18 @@ function calculateWomenAverageAge(people, withChildren) {
  * @return {number}
  */
 function calculateAverageAgeDiff(people, onlyWithSon) {
-  // write code here
+  const filterKid = people
+    .filter(person => people.find(mother => person.mother === mother.name));
```

Comment:
Method `some` would be better use for this purpose instead of `find`, because this way as I understand you need only boolean, not entire person object.